pkg/virt-controller/watch/drain/evacuation/evacuation.go

```diff
@@ -180,7 +180,7 @@ func (c *EvacuationController) addMigration(obj interface{}) {
 	node := ""
 
 	// only observe the migration expectation if our controller created it
-	key, ok := migration.Annotations[virtv1.EvacuationMigrationAnnotation]
+	key, ok := migration.Labels[virtv1.EvacuationMigrationLabel]
```

Comment:
You can leave this as an annotation. For the controller it does not really make a difference on which property you filter inside memory.

Reply:
I thought about this but ended up thinking that it'd be nice having a label instead, mainly for looking at things rapidly when using kubectl or to use label selectors in general (if anyone needs that).

But if you are of the opinion that we don't have anyone interested and we shouldn't be introducing a new label I won't be able to bring any justification to the table other than what I just mentioned :)

wdyt?

Reply:
Ok, I just changed it to use the annotation, I just don't have any arguments for a label :)